analysis/paper.Rmd

```diff
@@ -28,7 +28,7 @@ knitr::opts_chunk$set(echo = TRUE)
 # Methods
 
 ## Overview
-The original model was implemented in R (v3.5.2) [team2013r] with web application (https://johnquattrochi.com/bias/) to show that indirect method such as surveying can underestimate U5M by 0-41% in populations where 0-40% of the people are affected by HIV/AIDS. The model was written using the R language [team2013r] and some of the main packages used were tidyverse (v1.2.1) and foreach (v1.4.4). The original implementation utilized the doReids package (v1.1.1) that offers a back-end server for parallel processing that was not adopted in our replication due to the limitation in resources. In order to complete the simulation for constructing the model in a reasonable time frame without parallel computing, we reduced the number of populations used in the simulation input from 4480 to 1000. The code provided the author on Github (https://github.com/jquattro/hiv-childmort-bias) was mostly well-organized and documented. However, we discovered some mistakes regarding output inconsistency and wrong variable names. Despite the debugging process, we were able to replicate the implementation.
+The original model was implemented in R (v3.5.2) [team2013r] with web application (https://johnquattrochi.com/bias/) to show that indirect method such as surveying can underestimate U5M by 0-41% in populations where 0-40% of the people are affected by HIV/AIDS. The model was written using the R language [team2013r] and some of the main packages used were tidyverse (v1.2.1) and foreach (v1.4.4). The original implementation utilized the doReids package (v1.1.1) that launches a back-end server in Docker for parallel processing that was not adopted in our replication. The main limitation is the computing resource we had for this project within a short period of time. Based on experiments from multiple team members, finishing the first simulation step would require an 8-core CPU machine for 2 weeks. In order to complete the simulation for constructing the model in a reasonable time frame without parallel computing, we made a decision to reduce the sample size of our model -- the number of populations used in the simulation input has been reduced from 4480 to 1000. The code provided by the author on Github (https://github.com/jquattro/hiv-childmort-bias) was mostly well-organized and documented. However, we discovered some mistakes regarding output inconsistency and wrong variable names. Despite the debugging process, we were able to replicate the implementation.
```

Comment:
Please explain the experiments that were done to obtain the estimated time to finish the simulation. Also it wasn't a decision made by us to reduce the population size. I had to stop the simulation due to short of time.

Reply:
Simulation estimation part updated per request.